bundle/manifests/compliance.openshift.io_tailoredprofiles.yaml

```diff
@@ -57,6 +57,11 @@ spec:
                   description: RuleReferenceSpec specifies a rule to be selected/deselected,
                     as well as the reason why
                   properties:
+                    kind:
+                      description: |-
+                        Type of the rule reference, either "Rule" or "CustomRule"
+                        We will use "Rule" by default if not specified
```

Comment:
Nit: Suggest using passive voice here to be consistent with the rest of the properties.

`Type of rule, either "Rule" or "CustomRule". "Rule" is the default if not specified.`